### PR TITLE
Feature lwdev 3623 transfer eth trusted wallet

### PR DESCRIPTION
**Pull request by tarurar**

### CODE REVIEW

File: src/Lykke.Job.TransactionHandler/Queues/TransferQueue.cs [hunk 1]
```diff
@@ -158,18 +183,97 @@ public void Stop()
...
+            }
+            catch (Exception e)
+            {
+                await _log.WriteErrorAsync(nameof(TransferQueue), nameof(ProcessEthTransferTrustedWallet), e.Message, e);
+                ethError = $"{e.GetType()}\n{e.Message}";
+            }
+
+            if (!string.IsNullOrEmpty(ethError))
+            {
+                await _log.WriteErrorAsync(nameof(TransferQueue), nameof(ProcessEthTransferTrustedWallet), ethError,
```
Comment (polonzer) [inline src/Lykke.Job.TransactionHandler/Queues/TransferQueue.cs]: This is a repeating of line 266. The only way we got into this 'if' section if there was an exception a few lines before.

File: src/Lykke.Job.TransactionHandler/Queues/TransferQueue.cs [hunk 2]
```diff
@@ -158,18 +183,97 @@ public void Stop()
...
+                    null);
+
+                return false;
```
Comment (polonzer) [inline src/Lykke.Job.TransactionHandler/Queues/TransferQueue.cs]: Mixed error handling logic in lines 256-276. We should either save error context and process it in the end of function, or process directly and return result.

File: src/Lykke.Job.TransactionHandler/Queues/TransferQueue.cs (hunk 1 above)
Comment (tarurar) [reply src/Lykke.Job.TransactionHandler/Queues/TransferQueue.cs]: Correct, here is some strange error handling logic, fixed.

File: src/Lykke.Job.TransactionHandler/Queues/TransferQueue.cs (hunk 2 above)
Comment (tarurar) [reply src/Lykke.Job.TransactionHandler/Queues/TransferQueue.cs]: Fixed

**Pull request merged**